tpcx_bb/xbb_tools/utils.py

```diff
@@ -406,6 +406,11 @@ def get_tpcxbb_argparser_commandline_args():
     )
 
     args = parser.parse_args()
+
+    # ensure data_dir argument includes a trailing slash
```

Comment:
We can add this for now, but the right solution is to instead use Path objects or `os.path.join` in the Reader subclasses. When we do that, we should remove this.